pkg/go-runhcs/runhcs.go

```diff
@@ -5,6 +5,7 @@ package runhcs
 import (
 	"bytes"
 	"context"
+	"errors"
```

Comment:
What is this package even used for anymore? I thought runhcs was dead? Arent we only using shim for containerd these days?

Reply:
Looks like it's mostly used in tests for runhcs, which we still use to create scratch disks unfortunately :/

Reply:
OMG. Create-scratch. What's LCOW 🙄 ? LOL. Ok fine